molecule/shell.py

```diff
@@ -46,41 +46,28 @@ def _get_ansible_version():  # pragma: no cover
     return ansible.__version__
 
 
-def _supported_python2_version():  # pragma: no cover
-    return _get_python_version()[:2] == (2, 7)
+def _supported_python_version():  # pragma: no cover
+    py_version = _get_python_version()
```

Comment:
Just use
```suggestion
    py_version = sys.version_info
```

Having a separate one-liner doesn't really make it better since it's just a proxy for accessing one value.